### PR TITLE
Avoiding metadiscourse marker and empty words

### PR DESCRIPTION
**Pull request by msrahman07**
"This project is very early phase of experiment" --> "This project is an early phase of the experiment"
Line 45: Metadiscourse marker "However" not needed
Line 51: "And almost" --> empty words
Eliminated "But" "And" where sentences started with these

### CODE REVIEW

**Comment by rhysd** [comment]
Well, I'm not an native English user. Your fixes would correct. Let me ask one question. What tool did you use?

**Comment by msrahman07** [comment]
Thank you for replying. I didn’t use any tool! I am taking a course on Technical writing and editing where I am learning to write more clearly and simply for the readers. 
This pull request is a part of an assignment and I need this to be accepted and merged. 
Thank you again

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ sources into WebAssembly.
 
 ![Build Process](./wasm-readme-images/build-process.png)
 
-WebAssembly frontend for Vim is implemented as a new GUI frontend.  C sources are
+WebAssembly frontend for Vim is implemented as a new GUI frontend. C sources are
```
Comment (rhysd) [inline README.md]: Two spaces after period are rule of [original README](https://github.com/vim/vim/blob/master/README.md). I don't want to mix multiple conflicting rules in one README file. So please keep two spaces.

File: README.md [hunk 2]
```diff
@@ -41,14 +41,14 @@ I modified `configure` script to ignore the terminal library check.  It's OK sin
...
-some features are not supported by emscripten. I added many `#ifdef FEAT_GUI_WASM`
+emscripten provides Unix-like environment. So `os_unix.c` can support Wasm. Some 
+features are not supported by emscripten. I added many `#ifdef FEAT_GUI_WASM`
 guards to disable features which cannot be supported by Wasm (i.e. `fork (2)`
 support, PTY support, signal handlers are stubbed, ...etc).
 
 I created `gui_wasm.c` heavily referencing `gui_mac.c` and `gui_w32.c`. Event loop
 (`gui_mch_update()` and `gui_mch_wait_for_chars()`) is simply implemented with
-`sleep()`. And almost all UI rendering events arer passed to JavaScript layer
+`sleep()`. All UI rendering events are passed to JavaScript layer
```
Comment (rhysd) [inline README.md]: Could you update this line as follows? I think original one no longer represents current architecture correctly.

```markdown
`sleep()`.  Rendering UI to `<canvas>` is done in JavaScript layer
```

Could you also consider that max number of characters per one line is 80?

**Comment by rhysd** [comment]
Oh, so you did proofreading for this README.md. It's helpful. Let me review your changes.

**Review by rhysd — CHANGES_REQUESTED** [review]

**Comment by rhysd** [comment]
Please rebase this branch on `wasm` branch. I updated `wasm` branch to the latest upstream.

**Comment by rhysd** [comment]
@msrahman07 ping

**Comment by rhysd** [comment]
I'm closing this as no response. If you're still willing to contribute this, please ping me.

**Pull request closed without merging**